### ISSUE TITLE
[Pre-release] Deploy only content for dev

### ISSUE THREAD

**Issue by cvalarida** [department-of-veterans-affairs/va.gov-team#7971]
## Problem Statement
We need to deploy the content from the content build repository to the new, unused dedicated content S3 bucket

---
## Tasks
- [ ] Create a content deployment script in the `devops` repo
    - Similar to the `vets-website` deployment script, but omits application-related build pieces
       - I believe this is where the [current `vets-website` deploy script](https://github.com/department-of-veterans-affairs/devops/tree/master/ansible/deployment/roles/deploy-vets-website/)
- [ ] Set up Jenkins job to deploy content

## Acceptance Criteria
- [ ] The Jenkins job successfully builds the static content files and deploys them to the appropriate environments

## Questions
- How do environments play into this?
  - When do we deploy to which environments?
- How do we validate changes to the build code without deploying?
  - Need to make sure the actual deployment is deliberate

**Comment by billfienberg** [comment]
Related ticket #10001

**Comment by timwright12** [comment]
[View working branch (feature/7971-deploy-only-content-for-dev)](https://github.com/department-of-veterans-affairs/content-build/tree/feature/7971-deploy-only-content-for-dev)

**Comment by Clark-W-Griswold** [comment]
Open PR here:
https://github.com/department-of-veterans-affairs/devops/pull/8218

**Comment by billfienberg** [comment]
Current status: Implementing suggestion from [PR comment](https://github.com/department-of-veterans-affairs/devops/pull/8218#issuecomment-753017118)

**Comment by meganhkelley** [comment]
@dginther is working on investigating issues with the ansible playbook

Bill's PR calls ansible playbook, demian trying to run that ansible playbook locally but it wasn't working as expected — helping us debug

Related thread: https://dsva.slack.com/archives/CQH357ZTP/p1610145339345800

**Comment by timwright12** [comment]
Opened a new PR for this because of some uncomfortable merge conflict with the older stale one https://github.com/department-of-veterans-affairs/devops/pull/8534

**Comment by timwright12** [comment]
Need ops to set up Jenkins job here, reaching out today

**Comment by timwright12** [comment]
This ticket is waiting on https://app.zenhub.com/workspaces/vsp---frontend-tools-5fc9325744944e0015ed1861/issues/department-of-veterans-affairs/va.gov-team/11914 to complete

**Comment by timwright12** [comment]
content-build is now live on dev.va.gov
